PontoFacil/PontoFacil/Views/HistoryPage.xaml

```diff
@@ -62,52 +62,51 @@
+                                    DisplayMode="Month" x:Uid="ChooseAnEndDate"
                                     HorizontalAlignment="Stretch" Date="{Binding EndDate, Mode=TwoWay}"/>
             </StackPanel>
-            <Button x:Name="btnShowHistory" Content="Show History" Margin="10,25,0,0" Command="{Binding ShowFreeHistoryCommand}"></Button>
+            <Button x:Name="btnShowHistory" x:Uid="ShowHistory" Margin="10,25,0,0" Command="{Binding ShowFreeHistoryCommand}"></Button>
         </StackPanel>
 
-        <TextBlock Grid.Row="2" Text="History" FontSize="20" FontWeight="Bold" Margin="0,20,0,0"></TextBlock>
+        <TextBlock Grid.Row="2" x:Uid="History" FontSize="20" FontWeight="Bold" Margin="0,20,0,0"></TextBlock>
 
         <Border x:Name="historyListBorder" BorderBrush="Aqua" BorderThickness="1" Grid.Row="3" Margin="0,10,0,0" VerticalAlignment="Top">
             <StackPanel>
                 <StackPanel Orientation="Horizontal">
-                    <TextBlock Text="Date" Width="60" Style="{StaticResource HistoryTitle}"></TextBlock>
-                    <TextBlock Text="Start Time" Style="{StaticResource HistoryTitle}"></TextBlock>
-                    <TextBlock Text="End Time" Style="{StaticResource HistoryTitle}"></TextBlock>
-                    <TextBlock Text="Start Time" Style="{StaticResource HistoryTitle}"></TextBlock>
-                    <TextBlock Text="End Time" Style="{StaticResource HistoryTitle}"></TextBlock>
-                    <TextBlock Text="Worked Hours" Style="{StaticResource HistoryTitle}"></TextBlock>
-                    <TextBlock Text="Overtime Hours" Style="{StaticResource HistoryTitle}"></TextBlock>
-                    <TextBlock Text="Total Hours" Style="{StaticResource HistoryTitle}"></TextBlock>
-                    <TextBlock Text="Abono" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="Date" Width="60" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="StartTime" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="EndTime" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="StartTime" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="EndTime" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="WorkedHours" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="OvertimeHours" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="TotalHours" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="Waiver" Style="{StaticResource HistoryTitle}"></TextBlock>
                 </StackPanel>
                 <ListView x:Name="historyList" Margin="0,20,0,0" Height="auto" MaxHeight="700" ScrollViewer.VerticalScrollBarVisibility="auto"
```

Comment:
Extract the style of this ListView to the Page Resouce.

Reply:
Done @thiago-rocha ...Ready to review.